widget/edit.controller.js

```diff
@@ -1,88 +1,616 @@
 /* Copyright start
   MIT License
-  Copyright (c) 2023 Fortinet Inc
+  Copyright (c) 2024 Fortinet Inc
   Copyright end */
 'use strict';
 (function () {
   angular
     .module('cybersponse')
-    .controller('editSocOverviewSankey100Ctrl', editSocOverviewSankey100Ctrl);
+    .controller('editSocOverviewSankey200Ctrl', editSocOverviewSankey200Ctrl);
 
-  editSocOverviewSankey100Ctrl.$inject = ['$scope', '$uibModalInstance', 'config', 'appModulesService', 'Entity'];
+  editSocOverviewSankey200Ctrl.$inject = ['$scope', '$uibModalInstance', 'config', 'appModulesService', 'Entity', 'modelMetadatasService', 'widgetUtilityService', '$timeout'];
 
-  function editSocOverviewSankey100Ctrl($scope, $uibModalInstance, config, appModulesService, Entity) {
-    $scope.cancel = cancel;
-    $scope.save = save;
+  function socOverviewSankey200Ctrl($scope, $rootScope, config, $http, $q, Query, API, _, currentDateMinusService, PagedCollection, widgetUtilityService) {
```

Comment:
The function is replaced with a view controller.
Injected data is mismatched and the name is wrong.